### ISSUE TITLE
Set development mode and activate all debug logs

### ISSUE THREAD

**Issue by brunomacf** [magneticio/vamp-docker-images#38]
At now all logs are disabled by default and this is no satisfactory for work with vamp-docker-images as a development environment for all vamp components. Consider to add a flag to sinalize development mode and activate all debug logs.

**Comment by olafmol** [comment]
@brunomacf is this specifically for the QuickStart package?

**Comment by brunomacf** [comment]
@olafmol I don't think so, since we can run clique-* packages independently i think we should implement this strategy in each package. By the way, since quickstart is build upon other clique-* modules it will inherit from it. What do you think? Did i understand you right?

**Comment by olafmol** [comment]
i think it makes a lot of sense to have a single flag to enable/disable logging for all components. 👍

**Comment by brunomacf** [comment]
Me too... but consider this: mesos logs for example are disabled in clique-zookeeper-marathon image and quickstart module inherits from it. So, to enable logging through a DEV flag we should script this up in clique-zookeeper-marathon and then this will be available in quickstart image. The same is true for elasticsearch logs which are disabled (i.e., only error logs) in clique-base image and all other images inherits from it.
